packages/one/src/Root.tsx

```diff
@@ -75,41 +73,47 @@ export function Root(props: RootProps) {
     throw new Error(`No root component found`)
   }
 
+  const id = useId()
```

Comment:
I'm not sure if  `useId()` is collision safe in a server context, from my understanding its unique in a certain react tree (because it uses states in the parent). Maybe a simple uuid generator would be more appropriate.